src/hooks.ts

```diff
@@ -159,7 +159,7 @@ function subscribeToInitialization(
             clientReady: false,
             didTimeout: true,
           });
-          res.dataReadyPromise!.then(() => {
+          res?.dataReadyPromise?.then(() => {
```

Comment:
similar to previous comment

Reply:
dataReadyPromose can be null? If so, I do not see its handling.

Reply:
@jaeopt in client.ts L288, we assign `dataReadyPromise` but on L298 we return from the `USER_NOT_READY` without `dataReadyPromise` (`undefined`) which seems to be as designed on L42 earlier in the file.  

The optional chain `res.dataReadyPromise?.then(() => { ...` checks for this falsey scenario?  

Do see another solution?